src/cogs/pixel_art/layer.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+import time
+
+import disnake
+from disnake.ext import commands
+from PIL import Image
+
+from main import tracked_templates
+from cogs.pxls_template.progress import autocomplete_templates
+from utils.arguments_parser import MyParser
+from utils.pxls.template_manager import get_template_from_url, parse_template, layer
+from utils.setup import stats
+from utils.discord_utils import image_to_file
+
+
+class Layer(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot: commands.Bot = bot
+
+    @commands.slash_command(name="layer")
+    async def _reduce(
+        self,
+        inter: disnake.AppCmdInter,
+        first_template: str = commands.Param(autocomplete=autocomplete_templates),
+        second_template: str = commands.Param(autocomplete=autocomplete_templates),
+        third_template: str = commands.Param(
+            default=None, autocomplete=autocomplete_templates
+        ),
+        fourth_template: str = commands.Param(
+            default=None, autocomplete=autocomplete_templates
+        ),
+        fifth_template: str = commands.Param(
+            default=None, autocomplete=autocomplete_templates
```

Comment:
That's fair, but we'll lose the autocomplete. Lmk what you prefer

Reply:
I prefer all in one option yeah, I could work out a multiple-template-names autocomplete but it would just be a mess if people are mixing template links and template names.